README.md

```diff
@@ -15,7 +16,36 @@ depend on the number of vertices, the number of edges, or both?
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
 
+## Answer
+A function that converts an adjacency list to an adjacency matrix's time complexity is $O(V*E)$:
+
+1. var V = adjList.length; $O(1)$
+
+2. var matrix = Array.from(Array(V), ()=>Array(V).fill(0)); $O(v^2)$,because this creates 2D matrix depend on number of vertices
+
+3. for (var i = 0; i < V; i++) {// two for loops, first for loop depend on vertices, $O(V)$
+
+4. for (var j of adjList[i])//This for of depend on edges, $O(E)$
```

Comment:
But that can only happen for one vertex, right? You can't iterate over all edges for each vertex.

Reply:
No, because I set V as adjList.length. It will go through every vertex once.

Reply:
Ok, I changed the markdown file. I think is the degree of a vertex.